src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtilsImpl.java

```diff
@@ -199,29 +196,30 @@ public void addFormatAndPhaseSet(TableRow row, Genotype genotype, VCFHeader vcfH
         continue; // We will set GT field in a separate "genotype" field in BQ row.
```

Comment:
nit: How about renaming this function to  `addFormat()` just to make it consistent with `addInfo()`?